ENIGMAsystem/SHELL/Audio_Systems/None/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+set(NLIB "noneLIB")
```

Comment:
I'm not a fan of this nameing AUDIO_SYS_NONE instead maybe at very least there should be a an underscore before lib